src/main/java/uk/ac/ebi/atlas/experimentpage/tsneplot/TSnePlotDao.java

```diff
@@ -18,11 +19,12 @@ public TSnePlotDao(NamedParameterJdbcTemplate namedParameterJdbcTemplate) {
 
     private static final String SELECT_T_SNE_PLOT_WITH_EXPRESSION_STATEMENT =
             "SELECT tsne.cell_id, tsne.x, tsne.y, analytics.expression_level " +
-            "FROM scxa_tsne AS tsne " +
-                "LEFT JOIN " +
-                "(SELECT * FROM scxa_analytics WHERE gene_id=:gene_id) AS analytics " +
-                "ON analytics.cell_id=tsne.cell_id AND analytics.experiment_accession=tsne.experiment_accession " +
-            "WHERE tsne.experiment_accession=:experiment_accession AND tsne.perplexity=:perplexity";
+                    "FROM scxa_tsne AS tsne " +
+                    "LEFT JOIN " +
+                    "(SELECT * FROM scxa_analytics WHERE gene_id=:gene_id) AS analytics " +
+                    "ON analytics.cell_id=tsne.cell_id AND analytics.experiment_accession=tsne.experiment_accession " +
```

Comment:
```suggestion
                    "ON analytics.cell_id=tsne.cell_id " +
```

Reply:
One thing @pcm32. The above query is not modified by me. This is the existing codebase. Do you want me to change it?. It may impact test cases.